content/en-us/resources/modules/event-sequencer.md

```diff
@@ -787,7 +787,7 @@ Schema.OnRun = function()
 		StartTimes = {5, 27.25},
 		OnStart = function(self)
 			-- Initialize temporary heartbeat connection
-			local tempConnection = RunService.Heartbeat:Connect(function()
+			local tempConnection = RunService.PostSimulation:Connect(function()
```

Comment:
Note: The comment should be consistent.

Also, See above comment.

Reply:
Wasn't sure if PostSimulation was intended to be completely replacing Heartbeat or not in the future.